canonicalwebteam/flask_base/observability.py

```diff
@@ -0,0 +1,126 @@
+import sys
```

Comment:
I suggest create a module called `opentelemetry` and it should contain files called `prometheus`, `tracing`, `metrics`..

Current files that should be moved to this module: `observability.py` and `metrics.py`.